style.css

```diff
@@ -0,0 +1,127 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Poppins", sans-serif;
+}
+
+header {
+  min-height: 100vh;
+  width: 100%;
+  position: relative;
+}
+
+nav {
+  position: fixed;
+  left: 0;
+  right: 0;
+  top: 0;
+  height: 72px;
+  padding: 0 10%;
+  background-color: white;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+@media (min-width: 576px) {
+  .navLinks .fa-bars {
+    display: none;
+  }
+}
+
+@media (max-width: 576px) {
+  .navLinks ul li {
+    display: none;
+  }
```

Comment:
- [OPTIONAL] I think media queries are not required at this point since we are using the mobile first approach, please kindly fix that.